dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -1,9 +1,6 @@
 -# Gamelab-only fields for level edit page on levelbuilder
 - content_for(:head) do
-  %script{src: webpack_asset_path('js/levels/editors/_gamelab.js')}
-
-%legend.control-legend.levelTypeHeader
-  Game Lab Options
```

Comment:
intentional that this header is being removed? here and below

Reply:
Yup - They are confusing because not all Game Lab options are really under this header